code/SiteConfig.php

```diff
@@ -284,31 +280,6 @@ public function getCMSFields()
         return $fields;
     }
 
-    /**
-     * Get the actions that are sent to the CMS.
-     *
-     * In your extensions: updateEditFormActions($actions)
-     *
-     * @return FieldList
-     */
-    public function getCMSActions()
-    {
-        if (Permission::check('ADMIN') || Permission::check('EDIT_SITECONFIG')) {
-            $actions = FieldList::create(
-                FormAction::create(
-                    'save_siteconfig',
-                    _t('SilverStripe\\CMS\\Controllers\\CMSMain.SAVE', 'Save')
-                )->addExtraClass('btn-primary font-icon-save')
-            );
-        } else {
-            $actions = FieldList::create();
-        }
-
-        $this->extend('updateCMSActions', $actions);
-
-        return $actions;
-    }
```

Comment:
Let the admin handle the actions. `LeftAndMain` automatically adds a `save` action.
This method exists on `DataObject` so we don't need to deprecate this.